CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ full changeset diff at the end of each section.
 
 Current Trunk
 -------------
+- Change constant values of some reference types in the C and JS APIs. This is
+  only observable if you hardcode specific values instead of calling the
+  relevant methods (like `BinaryenTypeDataref()`). (#4755)
```

Comment:
Nit: I don't think this needs to be called out in the changelog since users shouldn't be using the raw values. For example, the numbers changed when we made `extern` an alias for `any`, but we didn't add that to the changelog.

Reply:
Yeah, it's not strictly needed I guess. I imagine though maybe someone is using the raw values even though they shouldn't - maybe they don't know they shouldn't. Then if they see these docs maybe that gets them to fix it before running into any problems. Overall I think such "ABI" changes can't hurt in the changelog.